app/subscriber/src/features/my-reports/edit/content/stories/ContentForm.tsx

```diff
@@ -71,12 +155,17 @@ export const ContentForm: React.FC<IContentFormProps> = ({
           rows={1}
           disabled={disabled}
           onChange={(e) => {
+            if (!content || typeof content.id !== 'number') {
```

Comment:
What's the value of this error message that is repeated throughout this page?  If content doesn't exist why is the user editing the form?